packages/sanity/src/core/validation/util/createBatchedGetDocumentExists.ts

```diff
@@ -11,8 +11,8 @@ import {
   switchMap,
   finalize,
 } from 'rxjs'
+import {ConcurrencyLimiter} from '@sanity/util/ConcurrencyLimiter'
```

Comment:
this feels weird, is there a reason it can't be exported from root of the util? `import { ConcurrencyLimiter} from '@sanity/util'`

Reply:
It's a good question. I've tried to follow the existing conventions in `@sanity/util`. For example, [`createSafeJsonParser` and `legacyDateFormat` are both individual exports](https://github.com/sanity-io/sanity/blob/next/packages/%40sanity/util/package.json#L58-L79). There are [no root exports](https://github.com/sanity-io/sanity/blob/next/packages/%40sanity/util/exports/index.ts).

Reply:
interesting, maybe double check with test of the team make sure this is the direction we want to go with?

Reply:
Good point! I've renamed the package to `@sanity/util/concurrency-limiter`.